### ISSUE TITLE
Articles are no longer being updated with latest TT-RSS

### ISSUE THREAD

**Issue by abcminiuser** [nilsbraden/ttrss-reader-fork#319]
Running 1.89.0 (compiled from https://github.com/nilsbraden/ttrss-reader-fork/releases/tag/v1.89.0) with latest TT-RSS `HEAD` (https://tt-rss.org/gitlab/fox/tt-rss/commit/18186149495108becf8e07344cbafef9110b5aa3).

With this new TT-RSS build the reader app is no longer showing unread articles (or any new articles at all) after a fetch from the server. The relevant failure in the Android debug log is:

```
07-22 23:36:54.269 2900-3145/org.ttrssreader I/JSONConnector: {"show_content":"1","include_attachments":"1","op":"getHeadlines","skip":"0","feed_id":"-4","view_mode":"unread","is_cat":"1","limit":"200"}
07-22 23:36:54.352 2900-3145/org.ttrssreader E/JSONConnector: Input data could not be read: guid is not a constant in org.ttrssreader.model.pojos.Article$ArticleField (null)
                                                              java.lang.IllegalArgumentException: guid is not a constant in org.ttrssreader.model.pojos.Article$ArticleField
                                                                  at java.lang.Enum.valueOf(Enum.java:200)
                                                                  at org.ttrssreader.model.pojos.Article$ArticleField.valueOf(Article.java:63)
                                                                  at org.ttrssreader.net.JSONConnector.parseArticle(JSONConnector.java:594)
                                                                  at org.ttrssreader.net.JSONConnector.parseArticleArray(JSONConnector.java:561)
                                                                  at org.ttrssreader.net.JSONConnector.getHeadlines(JSONConnector.java:956)
                                                                  at org.ttrssreader.controllers.Data.cacheArticles(Data.java:123)
                                                                  at org.ttrssreader.gui.CategoryActivity$CategoryUpdater.doInBackground(CategoryActivity.java:253)
                                                                  at org.ttrssreader.gui.CategoryActivity$CategoryUpdater.doInBackground(CategoryActivity.java:227)
                                                                  at org.ttrssreader.utils.AsyncTask$2.call(AsyncTask.java:155)
                                                                  at java.util.concurrent.FutureTask.run(FutureTask.java:237)
                                                                  at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1113)
                                                                  at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:588)
                                                                  at java.lang.Thread.run(Thread.java:818)```

**Comment by iacchi** [comment]
This is also true with the version present in the Play store. Not sure if it's an update of TTRSS that broke everything, or a bug in this app.

**Comment by derastronaut** [comment]
I've the same issue. With the original Tiny Tiny RSS Reader-App it works. Just tested with the trial-version.

**Comment by Accel02** [comment]
I've the same issue.

**Comment by reibuehl** [comment]
Same here - since roughly beginning last week after updating TT-RSS for the first time in a while.

**Comment by abcminiuser** [comment]
This is fixed in #320, when the repo owner merges it and rebuilds.

**Comment by TwizzyDizzy** [comment]
Ah, good to see I'm not the only one. Strange thing here is: It occured after I recently reset the database at startup. No update of the app involved. Haven't been able to get it working again since that (delete data / cache / reinstall app).

Cheers
Thomas
